packages/amplify_authenticator/dartdoc_options.yaml

```diff
@@ -0,0 +1,5 @@
+dartdoc:
+  categories:
+    "Prebuilt Widgets":
```

Comment:
This adds a page that lists all the classes tagged with the "Prebuilt Widget" category. The markdown file is used as the header. See: https://pub.dev/packages/dartdoc#categories

My plan is to link to this from the UI docs site, so that we don't have to list them all out on the docs UI site as well.

This is what it will look like:
![Screen Shot 2022-02-17 at 4 08 08 PM](https://user-images.githubusercontent.com/20613561/154571417-07e55ad8-c906-4c4c-8a06-7d0ee589ff62.png)

Reply:
Sweet!!